subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/InstrumentationConfiguration.kt

```diff
@@ -51,7 +52,15 @@ open class InstrumentationConfiguration(val name: String) {
 
     var kubernetesNamespace = "android-emulator"
 
+    /**
+     * It must be a valid reportId.
+     * Get test runs from report by id. Filter already succeed or new tests.
+     * Stay only failed contained in report
+     */
+    var filterSucceedAndNewTestsByReport: String? = null
```

Comment:
Name is strange because of implicit behavior that we ignore tests haven't test run

Reply:
`filter` is ambiguous. Is it "skip" or "run only"?
Can we name this final action instead of "filter"?